lib/db/mysql/patches/patch-019-020.sql

```diff
@@ -0,0 +1,5 @@
+ALTER TABLE clients
+ADD COLUMN scopes VARCHAR(1024) AFTER trusted,
```

Comment:
Naming nit: should we call this something like "allowedScopes" to make it clearer what the values here signify?